Library/Formula/crystax-ndk.rb

```diff
@@ -17,20 +17,15 @@ class CrystaxNdk < Formula
   depends_on "android-sdk" => :recommended
 
   conflicts_with "android-ndk",
-    :because => "both install `ndk-build`, `ndk-gdb` and `ndk-stack` binaries"
+    :because => "both install `ndk-build`, `ndk-gdb`, `ndk-stack`, `ndk-depends` and `ndk-which` binaries"
 
   def install
     bin.mkpath
 
-    if MacOS.prefer_64_bit?
-      arch = :x86_64
-    else
-      arch = :x86
-    end
+    # Cleanup some files since "brew audit" complains they're linked against system OpenSSL
+    %w{_hashlib.so _ssl.so}.each { |file| rm_f Dir.glob("prebuilt/darwin-*/lib/python*/lib-dynload/#{file}") }
```

Comment:
Alright, if you're removing them upstream I don't mind jumping the shark a little bit. A note in-formula that the files are being removed upstream in the next patch release and this line is safe to remove then would be great.